pkg/apis/batch/validation/validation_test.go

```diff
@@ -2141,7 +2211,7 @@ func TestValidateJobUpdateStatus(t *testing.T) {
 	}
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
-			errs := ValidateJobUpdateStatus(&tc.update, &tc.old)
+			errs := ValidateJobUpdateStatus(&tc.update, &tc.old, tc.opts)
```

Comment:
I don't see new test cases here

Reply:
Because the tests require passing opts, which are derived from the oldJob and newJob. However, the opts are computed in strategy.go. So, the easiest was to move the tests into strategy_test.go. An alternative would be to move the function deriving opts into validation.go and export. However, I wasn't sure which way to go, so just adding the tests to strategy_test.go seemed easiest. Let me know how you imagine this, maybe it is something we could improve by Test Freeze.

Reply:
That sounds acceptable